src/common_components/header/HeaderButton.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+import { css } from "@styled-system/css";
+import Image from "next/image";
+import { FC, MouseEventHandler, ReactNode } from "react";
+
+export const HeaderButton: FC<{
+  text: ReactNode;
+  icon: string;
+  clickev: MouseEventHandler<HTMLButtonElement>;
+}> = ({ text, icon, clickev }) => (
+  <button
+    className={css({
+      cursor: "pointer",
+      fontSize: "sm",
```

Comment:
```suggestion
      fontSize: {base: "sm", lg: "md"},
```
モバイル版だけで字が小さくなるようにお願いします